src/opcuabridge/imageblock.h

```diff
@@ -0,0 +1,58 @@
+#ifndef OPCUABRIDGE_IMAGEBLOCK_H_
+#define OPCUABRIDGE_IMAGEBLOCK_H_
+
+#include "common.h"
+
+namespace opcuabridge {
+class ImageBlock {
```

Comment:
Is the intention to use ImageBlock to transfer the update to the client?

If so then I think this might cause problems down the line--the Yocto integration we have available today uses libostree to manage the updates, and libostree has a 'git/object' like representation of a file system image: https://ostree.readthedocs.io/en/latest/manual/introduction/

Reply:
Yes, ImageBlock is used to transfer the update.
I need to look closer.

Reply:
I see it like we need custom version of ostree_repo_pull so it can use OPC-UA bridge to transport update to the secondary side.